tests/contrib/psycopg/test_psycopg.py

```diff
@@ -287,19 +285,6 @@ def test_composed_query(self):
             dict(name='postgres.query', resource=query.as_string(db)),
         )
 
-    def test_analytics_with_rate(self):
```

Comment:
why?

Reply:
`psycopg` like other `dbapi`-inherited integrations are enabled with environment variables for `dbapi` --- I haven't added integration-specific tests since `dbapi` has such tests and we'd be replicating them across these others as is.

Reply:
We probably should replicate them.

If we ever have to change `psycopg2` to not call one of the base classes from `dbapi2` then this feature will break and we likely won't know about it.